src/Ev.ServiceBus.HealthChecks/Ev.ServiceBus.HealthChecks.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net48;net5.0;netcoreapp2.1;netcoreapp3.1;netstandard2.0;netstandard2.1</TargetFrameworks>
+        <IsPackable>true</IsPackable>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <Description>This is a wrapper around Microsoft Azure Service Bus
+
+            Its goal to is make it the easiest possible to connect and handle an Azure ServiceBus resource (Queues, Topics or Subscriptions) inside ASP.NET Core.</Description>
```

Comment:
Could be change to describe the health check nuget

Reply:
Done